modelerfour/prenamer/prenamer.ts

```diff
@@ -133,7 +130,7 @@ export class PreNamer {
 
     // choice
     const choiceSchemaNames = new Set<string>();
-    for (const schema of values(this.codeModel.schemas.choices).toArray().sort(schemaNameComparer)) {
```

Comment:
Sorting schemas before deduplicating names causes changes in the output ordering.  I know you recommended that I do this @fearthecowboy, do you remember why?